pkg/util/net_test.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"testing"
+)
+
+const (
+	failed  = "\u2717"
+	succeed = "\u2713"
+)
+
+func TestRemovePortFromHost(t *testing.T) {
+	tests := []struct {
+		name     string
+		origHost string
+		expect   string
+	}{
+		{"Domain&Port", "localhost:8080", "localhost"},
+		{"Domain", "localhost", "localhost"},
```

Comment:
Can we add an addition full domain name test? Eg. www.kubernetes.test. As both domain and Domain&Port?

Reply:
Will add test cases for FQDN